primer_designer/views.py

```diff
@@ -114,21 +105,29 @@ def format_region(region):
     return cmd
 
 
-def call_primer_designer(request, regions_form, primer_path, cmd):
+def call_primer_designer(request, regions_form, cmd, output_path):
     """
     Calls primer designer with given formatted cmd string
 
     Args:
+        - request: the particular POST request from frontend
         - regions_form (django form): submitted form data
-        - primer_path (str): path to primer designer dir
         - cmd (str): region formatted str to pass to primer designer
-    Returns: None
+        - output_path: path to output directory /primer_designer/output
+
+    Returns: render HTML for output PDF download link
     """
-    primer_cmd = f'python3 {primer_path}/bin/primer_designer_region.py {cmd}'
+    primer_cmd = (
+            "docker run "
+            f"-v {PRIMER_DESIGNER_REF_PATH}:/reference_files "
+            f"-v {output_path}:/home/primer_designer/output "
+            f"--env REF_37=/reference_files/grch37/{REF_37} "
+            f"--env DBSNP_37=/reference_files/grch37/{DBSNP_37} "
+            "primer_designer "
```

Comment:
should the image name and tag be in the config in case it changes?

Reply:
made a new config PRIMER_IMAGE (this could be either the image name or container ID)